designsafe/static/scripts/data-depot/components/published/published-view.component.js

```diff
@@ -141,6 +149,10 @@ class PublishedViewCtrl {
             this.browser.project.report_set = this.browser.publication.reports;
         }
         if (this.project.value.projectType === 'field_recon'){
+            this.browser.project.mission_set = this.browser.publication.missions;
+            this.browser.project.collection_set = this.browser.publication.collections;
+            this.browser.project.analysis_set = this.browser.publication.analysiss;
```

Comment:
typo or intentional?
```suggestion
            this.browser.project.analysis_set = this.browser.publication.analysis;
```